doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_logging.rst

```diff
@@ -130,40 +119,42 @@ The following are the prefixes used to indicate the cores:
 Assisted multicore logging
 ==========================
 
-.. to review
-
-Assisted multicore logging uses dictionary-based logging to send messages without redundant strings to STM, and is based on the :ref:`logging_guide_dictionary` feature of the logging API provided by Zephyr.
-Instead of including the format strings in the log messages, it logs the addresses (message IDs) where the strings are stored, which reduces the size of the logging subsystem.
-If the data goes to the ETR buffer, the proxy core's responsibility is to dump this data.
-The host PC, equipped with a decoder tool, translates these addresses back into human-readable text using a JSON database generated during the build process.
+Assisted multicore logging uses dictionary-based logging to send messages without redundant strings to STM, and is based on the :ref:`zephyr:logging_guide_dictionary` feature of the logging API provided by Zephyr.
+For more information on assisted multicore logging, see :ref:`zephyr:logging_cs_stm`.
 
-When using logs, this method has the following advantages:
+Configuring logging
+-------------------
 
-* It reduces the size of the binary, as the strings used in the log messages are not stored in the binary itself.
-* It reduces the amount of data that needs to be sent to and processed by the application core, as the string formatting is offloaded to the host side.
+To show the logs for a given domain in the UART output, you must enable the STM for both the application core (as the ETR buffer processing proxy) and that domain's CPU.
+To do so, use the :ref:`zephyr:nordic-log-stm-dict` snippet when building the application for the related cores.
+For example, the following command uses the snippet when building for the application core::
```

Comment:
There is sample for STM at https://github.com/zephyrproject-rtos/zephyr/tree/main/samples/boards/nordic/coresight_stm
maybe it can be linked here?
Maybe this sample require some doc/readme too?

Reply:
Moreover, this doc provides some example stm output but not stating from which application/code. How user can achieve such logs.
Thus would be great it IMHO better to show logs from sample.

Reply:
> There is sample for STM at https://github.com/zephyrproject-rtos/zephyr/tree/main/samples/boards/nordic/coresight_stm maybe it can be linked here? Maybe this sample require some doc/readme too?

That's merged in Zephyr. Linking could be done here if considered relevant by @nordic-krch, however, documentation cannot be done upstream before CS2. This would be a task for a future release, if we actually support that sample. @nordic-krch Any input here?

Reply:
> Moreover, this doc provides some example stm output but not stating from which application/code. How user can achieve such logs. Thus would be great it IMHO better to show logs from sample.

@nordic-krch Any input here?